<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>900</width>
    <height>400</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="minimumSize">
   <size>
    <width>600</width>
    <height>400</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>900</width>
    <height>400</height>
   </size>
  </property>
  <property name="cursor">
   <cursorShape>ArrowCursor</cursorShape>
  </property>
  <property name="windowTitle">
   <string>Currency converter</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QHBoxLayout" name="horizontalLayout_3">
    <item>
     <layout class="QVBoxLayout" name="verticalLayout_2">
      <item>
       <layout class="QVBoxLayout" name="verticalLayout">
        <item>
         <layout class="QHBoxLayout" name="horizontalLayout">
          <item>
           <widget class="QLabel" name="from">
            <property name="font">
             <font>
              <family>Yu Gothic UI Semibold</family>
              <pointsize>18</pointsize>
             </font>
            </property>
            <property name="cursor">
             <cursorShape>ArrowCursor</cursorShape>
            </property>
            <property name="styleSheet">
             <string notr="true">color: #ffffff;</string>
            </property>
            <property name="text">
             <string>From:</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QLineEdit" name="input">
            <property name="font">
             <font>
              <family>Yu Gothic UI</family>
              <pointsize>9</pointsize>
             </font>
            </property>
            <property name="text">
             <string>1</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QComboBox" name="currencyFrom"/>
          </item>
         </layout>
        </item>
        <item>
         <layout class="QHBoxLayout" name="horizontalLayout_2">
          <item>
           <widget class="QLabel" name="to">
            <property name="font">
             <font>
              <family>Yu Gothic UI Semibold</family>
              <pointsize>18</pointsize>
             </font>
            </property>
            <property name="cursor">
             <cursorShape>ArrowCursor</cursorShape>
            </property>
            <property name="styleSheet">
             <string notr="true">color: #ffffff;</string>
            </property>
            <property name="text">
             <string>To:</string>
            </property>
           </widget>
          </item>
          <item>
           <spacer name="horizontalSpacer">
            <property name="orientation">
             <enum>Qt::Horizontal</enum>
            </property>
            <property name="sizeType">
             <enum>QSizePolicy::Preferred</enum>
            </property>
            <property name="sizeHint" stdset="0">
             <size>
              <width>25</width>
              <height>20</height>
             </size>
            </property>
           </spacer>
          </item>
          <item>
           <widget class="QLineEdit" name="output">
            <property name="font">
             <font>
              <family>Yu Gothic UI</family>
              <pointsize>9</pointsize>
             </font>
            </property>
            <property name="cursor">
             <cursorShape>ForbiddenCursor</cursorShape>
            </property>
            <property name="text">
             <string>1</string>
            </property>
            <property name="readOnly">
             <bool>true</bool>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QComboBox" name="currencyTo"/>
          </item>
         </layout>
        </item>
        <item>
         <spacer name="verticalSpacerBetweenDateAndCurrency">
          <property name="orientation">
           <enum>Qt::Vertical</enum>
          </property>
          <property name="sizeType">
           <enum>QSizePolicy::Preferred</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>20</width>
            <height>21</height>
           </size>
          </property>
         </spacer>
        </item>
        <item>
         <layout class="QGridLayout" name="gridLayout">
          <item row="8" column="2">
           <widget class="QDateEdit" name="rangeBeginDateEdit"/>
          </item>
          <item row="0" column="0">
           <widget class="QRadioButton" name="todayRadioButton">
            <property name="font">
             <font>
              <family>Yu Gothic UI Semibold</family>
              <pointsize>14</pointsize>
             </font>
            </property>
            <property name="styleSheet">
             <string notr="true">color: #ffffff;</string>
            </property>
            <property name="text">
             <string>Today</string>
            </property>
           </widget>
          </item>
          <item row="2" column="2">
           <widget class="QDateEdit" name="singleDateEdit"/>
          </item>
          <item row="2" column="0">
           <widget class="QRadioButton" name="singleDateRadioButton">
            <property name="font">
             <font>
              <family>Yu Gothic UI Semibold</family>
              <pointsize>14</pointsize>
             </font>
            </property>
            <property name="styleSheet">
             <string notr="true">color: #ffffff;</string>
            </property>
            <property name="text">
             <string>Single date</string>
            </property>
           </widget>
          </item>
          <item row="8" column="4">
           <widget class="QDateEdit" name="rangeEndDateEdit"/>
          </item>
          <item row="8" column="0">
           <widget class="QCheckBox" name="rangeDateCheckBox">
            <property name="font">
             <font>
              <family>Yu Gothic UI Semibold</family>
              <pointsize>14</pointsize>
             </font>
            </property>
            <property name="styleSheet">
             <string notr="true">color: #fff;</string>
            </property>
            <property name="text">
             <string>Date range</string>
            </property>
            <property name="checked">
             <bool>false</bool>
            </property>
           </widget>
          </item>
         </layout>
        </item>
        <item>
         <spacer name="verticalSpacerBetweenDateAndButton">
          <property name="orientation">
           <enum>Qt::Vertical</enum>
          </property>
          <property name="sizeType">
           <enum>QSizePolicy::Preferred</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>20</width>
            <height>6</height>
           </size>
          </property>
         </spacer>
        </item>
       </layout>
      </item>
      <item>
       <widget class="QPushButton" name="convertButton">
        <property name="font">
         <font>
          <family>Yu Gothic UI Semibold</family>
          <pointsize>16</pointsize>
         </font>
        </property>
        <property name="text">
         <string>Convert</string>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item>
     <widget class="QCustomPlot" name="graphic" native="true">
      <property name="enabled">
       <bool>true</bool>
      </property>
      <property name="minimumSize">
       <size>
        <width>300</width>
        <height>375</height>
       </size>
      </property>
      <property name="maximumSize">
       <size>
        <width>300</width>
        <height>375</height>
       </size>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
 </widget>
 <customwidgets>
  <customwidget>
   <class>QCustomPlot</class>
   <extends>QWidget</extends>
   <header>qcustomplot.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
